pkg/contexts/ocm/compdesc/versions/ocm.software/v3alpha1/zz_generated.deepcopy.go

```diff
@@ -103,8 +103,8 @@ func (in *Reference) DeepCopy() *Reference {
 func (in *Resource) DeepCopyInto(out *Resource) {
 	*out = *in
 	in.ElementMeta.DeepCopyInto(&out.ElementMeta)
-	if in.SourceRef != nil {
-		in, out := &in.SourceRef, &out.SourceRef
+	if in.SourceRefs != nil {
+		in, out := &in.SourceRefs, &out.SourceRefs
 		*out = make([]SourceRef, len(*in))
```

Comment:
doesn't it need to be:
```suggestion
		*out = make([]SourceRefs, len(*in))
```
???